theano/sandbox/cuda/tests/test_basic_ops.py

```diff
@@ -1190,8 +1203,8 @@ def test_many_arg_elemwise():
                 args = [numpy.cast['float32'](rng.randn(*shapes))
                         for arg in xrange(0, num_args)]
 
-                symb_args = [theano.tensor.TensorType('float32',
-                                                      (False,)*nb_dim)()
+                symb_args = [theano.tensor.TensorType(
+                             'float32', (False,) * nb_dim)()
```

Comment:
this is the error generated by flake8 when I copy and paste your code:
"test_basic_ops.py:1207:34: E126 continuation line over-indented for hanging indent"